mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -439,6 +439,13 @@ func resourceContainerCluster() *schema.Resource {
 										Default:     "default",
 										Description: `The Google Cloud Platform Service Account to be used by the node VMs.`,
 									},
+									"image_type": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Default:     "cos_containerd",
+										Description: `The default image type used by NAP once a new node pool is being created.`,
+										ValidateFunc: validation.StringInSlice([]string{"cos_containerd", "cos", "ubuntu_containerd", "ubuntu"}, true),
```

Comment:
@mrvarmazyar I had a total lapse of thought here, I guess. I am so sorry, this is the validation. I was thinking `DiffSuppress`, I actually think the best thing we can do here is just force capitalization.
```suggestion
										ValidateFunc: validation.StringInSlice([]string{"COS_CONTAINERD", "COS", "UBUNTU_CONTAINERD", "UBUNTU"}, false),
```

Reply:
@megan07 Thank you for the help, I've learned a lot :) I just committed your suggestions and I think this is the final one ^_^